tests/migration/Core/BasicDataUntouchedTest.php

```diff
@@ -1,28 +0,0 @@
-<?php declare(strict_types=1);
-
-namespace Shopware\Tests\Migration\Core;
-
-use PHPUnit\Framework\Attributes\CoversNothing;
-use PHPUnit\Framework\TestCase;
-use Shopware\Core\Framework\Test\TestCaseBase\KernelLifecycleManager;
-use Shopware\Core\Framework\Util\Hasher;
-use Shopware\Core\Migration\V6_3\Migration1536233560BasicData;
-
-/**
- * @internal
- */
-#[CoversNothing]
-class BasicDataUntouchedTest extends TestCase
-{
-    public function testBasicDataUntouched(): void
-    {
-        $file = KernelLifecycleManager::getClassLoader()->findFile(Migration1536233560BasicData::class);
-        static::assertIsString($file);
-
-        static::assertSame(
-            '21052fc38d2f82c5b9056d4fdaf64541',
-            Hasher::hashFile($file),
-            'BasicData migration has changed. This is not allowed.'
```

Comment:
instead of having this test, we might want to have a danger rule, to check if the file was touched. so reviewers keep a special look at it. 

I mean this test was not written without purpose :grimacing: the data created in this migration should not be changed at all